spec/javascripts/models/beta/iteration_spec.js

```diff
@@ -1331,14 +1051,139 @@ describe("iteration", function() {
           const sprints = Iteration.groupBySprints(
             stories,
             project,
-            initialSprintNumber,
-            []
+            initialSprintNumber
           );
 
           expect(sprints[1].isFiller).toBeTruthy();
         });
       });
     });
+
+    describe('sprints point', () => {
+      const project = {
+        startDate: moment().format(),
+        iterationLength: 1,
+        currentSprintVelocity: 8
+      };
+
+      describe('when there is two feature stories with 2 and 3 points', () => {
+        const stories = [
+          {
+            id: 1,
+            position: "3.2",
+            state: "unstarted",
+            estimate: 2,
+            storyType: "feature"
+          },
+          {
+            id: 2,
+            position: "10",
+            state: "unstarted",
+            estimate: 3,
+            storyType: "feature"
+          }
+        ];
+
+        const sprints = Iteration.groupBySprints(
+          stories,
+          project,
+          undefined
+        );
+
+        it('returns 5', () => {
+          expect(sprints[0].points).toBe(5);
+        });
+      });
+
+      describe('when there is only one bug story', () => {
+        const stories = [
+          {
+            id: 1,
+            position: "3.2",
+            state: "unstarted",
+            estimate: "",
+            storyType: "bug"
+          }
+        ];
+
+        const sprints = Iteration.groupBySprints(
+          stories,
+          project,
+          undefined
+        );
+
+        it('returns 0', () => {
+          expect(sprints[0].points).toBe(0);
+        });
+      });
+
+      describe('when there is one bug and one feature with 2 points', () => {
+        const stories = [
+          {
+            id: 1,
+            position: "3.2",
+            state: "unstarted",
+            estimate: "",
+            storyType: "bug"
+          },
+          {
+            id: 2,
+            position: "4.2",
+            state: "unstarted",
+            estimate: 2,
+            storyType: "feature"
+          },
+        ];
+
+        const sprints = Iteration.groupBySprints(
+          stories,
+          project,
+          undefined
+        );
+
+        it('returns 2', () => {
+          expect(sprints[0].points).toBe(2);
+        });
+
+        it('returns a number', () => {
+          expect(typeof (sprints[0].points)).toBe('number');
+        });
+      });
+
+      describe('when there is two features one with 2 points and other with no estimate', () => {
+        const stories = [
+          {
+            id: 1,
+            position: "3.2",
+            state: "unstarted",
+            estimate: "",
+            storyType: "feature"
+          },
+          {
+            id: 2,
+            position: "4.2",
+            state: "unstarted",
+            estimate: 2,
+            storyType: "feature"
+          },
+        ];
+
+        const sprints = Iteration.groupBySprints(
```

Comment:
Avoid calling methods outside of `it` this way. Even if you have to make some duplication, always the logic you're inside the `it`

Reply:
Notice that it happens more times in this file, let's solve all of them :smile:

Reply:
Got it